lua/smart-splits/config.lua

```diff
@@ -8,24 +9,18 @@ local M = {
     'NvimTree',
   },
   move_cursor_same_row = false,
-  resize_mode_quit_key = '<ESC>',
-  resize_mode_silent = false,
+  resize_mode = {
+    quit_key = '<ESC>',
+    silent = false,
+    hooks = {
+      on_enter = nil,
+      on_leave = nil,
+    },
+  },
```

Comment:
this change is going to break people's configurations unless we handle the old style configurations.

Please update the `setup` method to check for the old values and copy them to the right place, so that we don't break people's configurations.

Please also update the README.md with the new options.